admob-latest/src/main/java/com/rakuten/tech/mobile/admob/AdMobDisplayer20.kt

```diff
@@ -0,0 +1,222 @@
+package com.rakuten.tech.mobile.admob
```

Comment:
I don't think this package name is correct: it should something like `com.rakuten.tech.mobile.miniapp.ads`.

Reply:
okay..updated it @corycaywood